PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java

```diff
@@ -104,10 +105,9 @@ public static boolean isValidStagingFolderPath(String stagingFolderPath) {
     }
 
     public static Module[] listFunctionModules(Project project) {
-        MavenProjectsManager mpm = MavenProjectsManager.getInstance(project);
         final Module[] modules = ModuleManager.getInstance(project).getModules();
         return Arrays.stream(modules).filter(m -> {
-            if (mpm.findProject(m) == null) {
+            if (isModuleInTestScope(m)) {
```

Comment:
Need test whether module is null here as `GlobalSearchScope.moduleWithLibrariesScope` didn't accept null

Reply:
ModuleManager.getInstance(project).getModules() will not return null modules.